zebra/redistribute.c

```diff
@@ -770,6 +770,14 @@ void zebra_import_table_rm_update(const char *rmap)
 				continue;
 			table = zebra_vrf_table_with_table_id(afi, SAFI_UNICAST,
 							      i, VRF_DEFAULT);
+			if (!table) {
+				if (IS_ZEBRA_DEBUG_RIB_DETAILED)
+					zlog_debug(
+						"\t%s: Table id=%d not found",
+						__PRETTY_FUNCTION__, i);
```

Comment:
use __func__

Reply:
I like __PRETTY_FUNCTION__

Reply:
I was just following the way it does it in `zebra_rib.c`

Reply:
Would you prefer it with __func__ and no `\t`?

Reply:
@donaldsharp its literally the same thing but longer

kill the tab I think